KNOWNBUG
main.cpp

^main\.cpp:26:  Could not find function header comment for foo  \[readability/function_comment\] \[4\]
^# Total errors found: 1$
^EXIT=1$
^SIGNAL=0$
--
--
The checking code in cpplint.py was broken in 7302e56a348, where a wrong indent
caused the check to never take place. That's fine, however, as we don't use
these function headers anymore.
